ambry-frontend/src/test/java/com.github.ambry.frontend/TtlUpdateHandlerTest.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.frontend;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.account.Account;
+import com.github.ambry.account.Container;
+import com.github.ambry.account.InMemAccountService;
+import com.github.ambry.account.InMemAccountServiceFactory;
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.clustermap.MockClusterMap;
+import com.github.ambry.commons.ByteBufferReadableStreamChannel;
+import com.github.ambry.config.FrontendConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.messageformat.BlobProperties;
+import com.github.ambry.rest.MockRestRequest;
+import com.github.ambry.rest.MockRestResponseChannel;
+import com.github.ambry.rest.ResponseStatus;
+import com.github.ambry.rest.RestRequest;
+import com.github.ambry.rest.RestResponseChannel;
+import com.github.ambry.rest.RestServiceErrorCode;
+import com.github.ambry.rest.RestServiceException;
+import com.github.ambry.rest.RestUtils;
+import com.github.ambry.router.GetBlobOptionsBuilder;
+import com.github.ambry.router.GetBlobResult;
+import com.github.ambry.router.InMemoryRouter;
+import com.github.ambry.router.PutBlobOptionsBuilder;
+import com.github.ambry.router.ReadableStreamChannel;
+import com.github.ambry.utils.TestUtils;
+import com.github.ambry.utils.Utils;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Properties;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+
+/**
+ * Tests for {@link TtlUpdateHandler}.
+ */
+public class TtlUpdateHandlerTest {
+  private static final InMemAccountService ACCOUNT_SERVICE =
+      new InMemAccountServiceFactory(false, true).getAccountService();
+  private static final Account REF_ACCOUNT = ACCOUNT_SERVICE.createAndAddRandomAccount();
+  private static final Container REF_CONTAINER = REF_ACCOUNT.getContainerById(Container.DEFAULT_PRIVATE_CONTAINER_ID);
+  private static final ClusterMap CLUSTER_MAP;
+  private static final String SERVICE_ID = "TtlUpdateHandlerTest";
+  private static final long TTL_SECS = 7200;
```

Comment:
There is `TTL_SECS` in a bunch of test files.  Maybe we can add that into a TestUtil file and thus `static` makes more sense.

Reply:
Added to `com.github.ambry.utils.TestUtils`